src/main/java/org/opentripplanner/graph_builder/module/osm/OsmModule.java

```diff
@@ -252,7 +259,9 @@ private void buildBasicGraph() {
         params.banDiscouragedBiking(),
         issueStore
       );
-      if (!OsmFilter.isWayRoutable(way) || permissions.allowsNothing()) continue;
+      if (
+        !OsmFilter.isWayRoutable(way) || permissions.allowsNothing() || way.hasTag("conveying")
```

Comment:
I think it's better to use the `isEscalator` method here as well from the `OSMWay` class. Also, wrap the continue with `{}` and put it on an own line (I know it was like this before but it shouldn't be).